static/app/components/events/eventCustomPerformanceMetrics.tsx

```diff
@@ -103,12 +111,65 @@ function EventCustomPerformanceMetric({
       )
     : value;
 
+  function generateLinkWithQuery(query: string) {
+    const eventView = EventView.fromSavedQuery({
+      query,
+      fields: [],
+      id: undefined,
+      name: '',
+      projects: [],
```

Comment:
This is more of an open UX question (probably don't even need to look into it till we get feedback), but when when clicking one of the options, I _might_ expect the filter to get added to the existing query (persist columns and projects that i've selected) I'm already looking at instead of putting me in a whole new view (ie fields: [] and projects: [])

Reply:
👍 agreed, ideally it should also preserve existing filters when linking back. 
I'll add this in a separate PR!